app/scripts/modules/core/src/pipeline/config/triggers/artifacts/expectedArtifact.html

```diff
@@ -0,0 +1,35 @@
+<div class="row">
+  <div class="col-md-12">
+    <div class="form-horizontal panel-pipeline-phase">
+      <div class="form-group row">
+        <div class="col-md-3">
+          Match against
+        </div>
+        <div class="col-md-1 col-md-offset-8">
+          <button class="btn-link" ng-click="ctrl.removeArtifact()">
+            <span class="glyphicon glyphicon-trash" uib-tooltip="Remove artifact"></span>
+            <span class="sr-only">Remove artifact</span>
+          </button>
+        </div>
+      </div>
+      <artifact artifact="ctrl.expectedArtifact.matchArtifact"></artifact>
+      If missing
+      <div class="form-group row">
+        <label class="col-md-3 sm-label-right">
+          Use Prior Execution
```

Comment:
I don’t remember - can these both be checked? If not, you should use a radio toggle here instead.

Reply:
Oh I just read what you said at the top. Some help text to describe the behavior would be nice.